src/EventExport/Command/ExportEvents.php

```diff
@@ -74,4 +79,22 @@ public function getInclude(): array
     {
         return $this->include;
     }
+
+    private function setSortOrder(SortOrder $sortOrder): void
```

Comment:
I would remove this private setter, there is no real added value.

Reply:
Done

[Replace sortOrder setters by property assignment](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/0029539335c3896a4b4030ccb4eb3d767e988d1c)